rdap/objects.py

```diff
@@ -124,3 +139,50 @@ def _parse(self):
             org_name=org_name,
             org_address=org_address,
         )
+
+
+class RdapDomain(RdapObject):
+    def __init__(self, data, rdapc=None):
+        super().__init__(data, rdapc)
+
+
+class RdapEntity(RdapObject):
+    def __init__(self, data, rdapc=None):
+        super().__init__(data, rdapc)
+
+    @property
+    def emails(self):
+        return self.parsed()['emails']
+
+    @property
+    def kind(self):
+        return self.parsed()['kind']
+
+    def _parse(self):
+        """ parses data into our format """
+        # emails done with a set to eat duplicates
+        emails = set()
+
+        vcard = self._parse_vcard(self._data)
+        emails |= vcard.get('emails', set())
+
+        name = vcard.get('fn', None)
+        address = vcard.get('adr', None)
+        kind = vcard.get('kind', None)
+
+        # WORKAROUND APNIC keeps org info in remarks
+        if 'apnic' in self._data.get('port43', None):
```

Comment:
this will error if port43 is not set, should default to a `""` i think ?

Reply:
you're right, I didn't mean to do the PR, I meant my mods, thanks :)